source/agora/consensus/Fee.d

```diff
@@ -388,7 +388,7 @@ public class FeeManager
 
             this.db.execute(
                 "REPLACE INTO accumulated_fees (fee, public_key) VALUES (?,?)",
-                this.accumulated_fees[stake.output.address], stake.output.address);
+                this.accumulated_fees[stake.output.address].tupleof[0], stake.output.address);
```

Comment:
Do you need the `tupleof` though ? Since there's a `toString` overload it should pick that up.

Reply:
But it uses `INTEGER` SQL type now.

Reply:
Right, it only uses them for string type.

Reply:
If not needed for `Amount` -> `INTEGER` I will remove in next PR.

Reply:
In fact it is removed already as I had to change the table again to store for each height.